src/misc/unicode.cpp

```diff
@@ -32,416 +32,618 @@
 
 CHECK_NARROWING();
 
-
 class Grapheme final {
 public:
-    Grapheme() = default;
-    Grapheme(const uint16_t code_point);
+	Grapheme() = default;
+	Grapheme(const uint16_t code_point);
 
-    bool IsEmpty() const;
-    bool IsValid() const;
-    bool HasMark() const;
-    uint16_t GetCodePoint() const;
+	bool IsEmpty() const;
+	bool IsValid() const;
+	bool HasMark() const;
+	uint16_t GetCodePoint() const;
```

Comment:
Ensure return types are always checked (otherwise the caller isn't using the function right):

```c++
	[[nodiscard]] bool IsEmpty() const;
	[[nodiscard]] bool IsValid() const;
	[[nodiscard]] bool HasMark() const;
	[[nodiscard]] uint16_t GetCodePoint() const;
```

Reply:
Ok, changed.